### ISSUE TITLE
contrib section docs should mention openssl-devel requirement

### ISSUE THREAD

**Issue by omiday** [ralphbean/bugwarrior#404]
Without `openssl-devel` package installed I was getting:

```
>>> bugwarrior (develop>) bugwarrior $ python setup.py develop
running develop
running egg_info
writing requirements to bugwarrior.egg-info/requires.txt
writing bugwarrior.egg-info/PKG-INFO
writing top-level names to bugwarrior.egg-info/top_level.txt
writing dependency_links to bugwarrior.egg-info/dependency_links.txt
writing entry points to bugwarrior.egg-info/entry_points.txt
reading manifest file 'bugwarrior.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*' under directory 'docs'
writing manifest file 'bugwarrior.egg-info/SOURCES.txt'
running build_ext
Creating /home/lelu/.virtualenvs/bugwarrior/lib/python2.7/site-packages/bugwarrior.egg-link (link to .)
bugwarrior 1.4.0 is already the active version in easy-install.pth
Installing bugwarrior-pull script to /home/lelu/.virtualenvs/bugwarrior/bin
Installing bugwarrior-uda script to /home/lelu/.virtualenvs/bugwarrior/bin
Installing bugwarrior-vault script to /home/lelu/.virtualenvs/bugwarrior/bin

Installed /home/lelu/work/bugwarrior
Processing dependencies for bugwarrior==1.4.0
Searching for cryptography
Reading https://pypi.python.org/simple/cryptography/
Downloading https://pypi.python.org/packages/03/1a/60984cb85cc38c4ebdfca27b32a6df6f1914959d8790f5a349608c78be61/cryptography-1.5.2.tar.gz#md5=f977a6ddb38b071213c0157fc77c62ab
Best match: cryptography 1.5.2
Processing cryptography-1.5.2.tar.gz
Writing /tmp/easy_install-9_uQmP/cryptography-1.5.2/setup.cfg
Running cryptography-1.5.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-9_uQmP/cryptography-1.5.2/egg-dist-tmp-d7lOEB
no previously-included directories found matching 'docs/_build'
warning: no previously-included files matching '*' found under directory 'vectors'
build/temp.linux-x86_64-2.7/_openssl.c:433:30: fatal error: openssl/opensslv.h: No such file or directory
#include <openssl/opensslv.h>
                            ^
compilation terminated.
error: Setup script exited with error: command 'gcc' failed with exit status 1
```

Thanks!

**Comment by coddingtonbear** [comment]
It looks like you don't have openssl libraries installed.  How you install those varies by OS, but if you're on linux, you could search for the package; it might be named something like `openssl-dev`.

**Comment by ryneeverett** [comment]
Looks like this was fixed in 8132b9658fad370f2cf358122942601c3882a47d.
